app/scripts/models/notifications.js

```diff
@@ -13,7 +13,8 @@ define([
   'use strict';
 
   var EVENTS = {
-    PROFILE_CHANGE: 'profile:change'
+    PROFILE_CHANGE: 'profile:change',
+    LOGOUT: 'fxaccounts:logout'
```

Comment:
> I wonder if we need to distinguish between "logout" and "delete". Listeners might want to react differently to the two.

I'm still wondering about this. Unless there's a legacy reason not to, it seems like we should be semantic here and let the client decide how it handles the notification.